collections/search/js/searchform.js

```diff
@@ -191,6 +192,15 @@ function addChip(element) {
       removeChip(inputChip);
     };
   }
+  let screenReaderSpan = document.createElement("span");
+  const dataChipText = element.getAttribute("data-chip");
+  const screenReaderText = dataChipText
+    ? dataChipText
+    : element?.text || "Unknown chip";
+  screenReaderSpan.textContent =
+    "Remove " + screenReaderText + " from search criteria";
```

Comment:
Internationalization of this will be nontrivial. This function lives in a javaScript file, and its parent methods addChip and updateChip don't interface directly with the PHP file that leverages this js file. `setSearchForm` , which calls updateChip, does interface directly. I can pass the translated strings to setSearchForm, BUT there are onChange listeners that also call updateChip, and those listeners DON'T have access to the variables passed to setSearchForm. So, essentially, any time new chips are made in the criteria section, they can't get access to the translated text; only those initialized on page load would have access to that text. Instead of doing that, I decided to hard-code the text here.

I suppose there is another solution to this, but it's not clear to me after about an hour of playing with this. Feels like killing a mosquito with a cannon at this point, since this text will only be seen by a screen reader, and our most pressing 508 compliance audience is American.

My vote is to leave this untranslated for now and fry bigger fish.